### PR TITLE
BB Point Bots Update

### PR DESCRIPTION
**Pull request by sohamdeogaonkar**

### CODE REVIEW

File: commands/_setAngryPoints.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  help: 
+  need_reply: 
+  auto_retry_time: 
+  folder: 
+  answer: 
+  keyboard: 
+  aliases: 
+CMD*/
+
+var AngryPointsdata = AdminPanel.getPanel("AngryPoints")
```
Comment (bots-business) [inline commands/_setAngryPoints.js]: please rename this to angryPointsPanel

Comment (bots-business) [reply commands/_setAngryPoints.js]: from AngryPointsdata

File: commands/Transfering/transfer.js [hunk 2]
```diff
@@ -36,8 +36,32 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(removalPoints) {
+  if (removalPoints > 0) {
+    return false
+  }
+  var angryPointsLimit = Libs.ResourcesLib.userRes("angryPointsLimit")
```
Comment (bots-business) [inline commands/Transfering/transfer.js:43]: angryPointsLimit - I think it still not good name

angryPointsMaxLimit - is more good

File: commands/Transfering/transfer.js [hunk 3]
```diff
@@ -36,8 +36,32 @@ function successMessage(res, amount){
...
+  if (-removalPoints > angryPointsLimit.value()) {
+    // Master can remove points without exceeding the limit
+    return false
+  }
+
+  var angryPoints = Libs.ResourcesLib.userRes("angryPoints")
```
Comment (bots-business) [inline commands/Transfering/transfer.js:49]: angryPointsHourlyLimit - please rename. It is Hourly limit

**Review by bots-business — CHANGES_REQUESTED** [review]

**Comment by sohamdeogaonkar** [comment]
image

admin panel setup js
image

**Comment by sohamdeogaonkar** [comment]
@bots-business ?

File: commands/Transfering/transfer.js (hunk 3 above)
Comment (bots-business) [reply commands/Transfering/transfer.js:49]: avaibleAngryPoints - please rename it

File: commands/_setAngryPoints.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+var angryPoints = Libs.ResourcesLib.anotherUserRes("angryPoints", masterId)
+angryPoints.growth.add({
+  value: parseInt(hourlyGrowth),
+  interval: sec_in_hr,
+  max: parseInt(maxGrowthLimit)
+})
+
+//making the admin panel empty so that bot admin can add more masters
+
+angryPointsPanel.fields[0].value = ""
```
Comment (bots-business) [inline commands/_setAngryPoints.js:38]: I think we do not need this clearing

**Review by bots-business — CHANGES_REQUESTED** [review]

**Review by bots-business — CHANGES_REQUESTED** [review]
need changes

**Pull request closed without merging**